org.nongnu.enigma.yml

```diff
@@ -80,9 +169,7 @@ modules:
         path: 0001-Invalid-pointer-access.patch
       - type: patch
         path: 0001-Fix-.desktop-to-success-the-desktop-filde-validate.patch
-      - type: file
-        path: org.nongnu.enigma.metainfo.xml
+      - type: patch
+        path: 0001-fix-appstream.patch
     post-install:
-      - install -d /app/share/icons/hicolor/128x128/apps/
-      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
-      - convert /app/share/icons/hicolor/48x48/apps/enigma.png -resize 128x128 /app/share/icons/hicolor/128x128/apps/enigma.png
```

Comment:
I think that's the reason of imagemagick in the package, isn't it?

Reply:
Nope, I saw the enigma makefile doing stuff during build, unrelated to this step